doc/march_packages/march_simulation.rst

```diff
@@ -8,53 +8,55 @@ Overview
 The march simulation package launches `Gazebo <http://gazebosim.org/>`_ and spawns the :ref:`URDF <robot-model-label>` of the |march|.
 Its goal is to provide an interface for our high-level code to call similar to that of the :ref:`march-hardware-interface-label`.
 
-As we only use Gazebo for actuating joints, using it is very simple.
-Please check :simulation:`march_world.launch <march_simulation/launch/march_world.launch>` to see how its used.
+Usage
+-----
 
-ROS Control
-^^^^^^^^^^^
-To integrate Gazebo with ROS Control, we highly recommend checking out the `Gazebo ros control tutorial <http://gazebosim.org/tutorials/?tut=ros_control>`_
-The :simulation:`config files <march_simulation/config>` contain example controller configuration.
+The simulation can be launched using
 
-Upload joint names
-^^^^^^^^^^^^^^^^^^
-
-The :simulation:`upload_joint_names <march_simulation/scripts/upload_joint_names>` script loads the URDF and uploads the joint names to the topic ``/march/joint_names``.
-This is used by the rest of the system to check which joints are active.
+.. code::
 
-Tutorials
----------
+  roslaunch march_launch march_simulation.launch
 
-Sending commands through the action topic
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Start the simulation with
+This will launch the simulation using the rviz GUI. The backpack is fixed in space so the exoskeleton will move
+similarly to airgaiting. It is also possible to groundgait in the simulation. We will need the Gazebo GUI for that,
+so that the exoskeleton can move relative to the world. The launch command becomes
 
 .. code::
 
-  roslaunch march_simulation march_world.launch
+  roslaunch march_launch march_simulation.launch rviz:=false gazebo_ui:=true ground_gait:=true
 
-Then send a command to the controller with
+One can add an obstacle in simulation by adding the "obstacle:=<<obstacle_name>>" to the command. The possible
```

Comment:
May be explain a bit more on obstacles, why these add some reference to the Cybathlon. For someone who does not know about the Cybathlon this seems a bit random I think.